web/source/osk/bannerManager.ts

```diff
@@ -103,6 +115,10 @@ namespace com.keyman.osk {
           break;
         case 'suggestion':
           this._setBanner(new SuggestionBanner());
+          let keyman = com.keyman.singleton;
+          let banner = this.activeBanner as SuggestionBanner;
+          keyman.modelManager['addEventListener']('invalidatesuggestions', banner.invalidateSuggestions.bind(this));
+          keyman.modelManager['addEventListener']('suggestionsready', banner.updateSuggestions.bind(this));
```

Comment:
It doesn't need to be done immediately, but it's worth noting that we'll need the ability to `'removeEventListener'` when the `SuggestionBanner` is swapped out for a different type of `Banner`.  That may need a bit more infrastructure; just noting it for future work.  (To do this, we'll need to maintain a single reference to a pre-bound version of the two listener methods.)